src/components/home/HomeHowItWorkSimple.vue

```diff
@@ -4,7 +4,7 @@
 
       <Card title="Bring your data">
         <h4>CMSs</h4>
-        <p>Any Headless CMS, Contentful, WordPress, Drupal, Sanity.io, etc.</p>
+        <p>Any Headless CMS, <a href="https://buttercms.com">ButterCMS</a>, Contentful, WordPress, Drupal, Sanity.io, etc.</p>
```

Comment:
Can we remove the link here? There are lots of companies mentioned on this page, so we should either be linking all of them or none of them.